examples/connection_pools/bootstrap.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
```

Comment:
Not sure where this file came from... I'll delete it since it doesn't appear to be useful in any way.

Reply:
This is used by the test runner to do any setting up necessary for the example to work properly. For instance, in the `todo` example, `bootstrap.sh` sets up the database.